pom.xml

```diff
@@ -244,6 +241,10 @@
         <dependency>
             <groupId>io.fabric8</groupId>
             <artifactId>kubernetes-server-mock</artifactId>
+        </dependency>
+		<dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk15on</artifactId>
         </dependency>
```

Comment:
Why do we need this? Is it a fabric8 dependency? I'm quite sure that we don't use it in the CM, could it be an option to cut it off the project?

Reply:
You are totally right, I have no idea why I ended up adding it, probably a wrong copy/paste. I just removed it and it runs locally as it should.